invenio_communities/views/communities.py

```diff
@@ -391,19 +410,22 @@ def communities_settings_privileges(pid_value, community, community_ui):
 
 
 @pass_community(serialize=True)
-def communities_settings_curation_policy(pid_value, community, community_ui):
-    """Community settings/curation-policy page."""
+def communities_settings_submission_policy(pid_value, community, community_ui):
+    """Community settings/submission-policy page."""
```

Comment:
```suggestion
    """Community settings/record-submission-policy page."""
```
depends on the decision to split or not the tabs